images/webhooks/src/validators/pvcMutation.go

```diff
@@ -0,0 +1,90 @@
+package validators
+
+import (
+	"context"
+	"github.com/slok/kubewebhook/v2/pkg/model"
+	kwhmutating "github.com/slok/kubewebhook/v2/pkg/webhook/mutating"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+)
+
+const annBetaStorageProvisioner = "volume.beta.kubernetes.io/storage-provisioner"
+const annStorageProvisioner = "volume.kubernetes.io/storage-provisioner"
+
+func PVCMutation(ctx context.Context, _ *model.AdmissionReview, obj metav1.Object) (*kwhmutating.MutatorResult, error) {
+	pod, ok := obj.(*corev1.Pod)
+	if !ok {
+		// If not a pod just continue the mutation chain(if there is one) and do nothing.
+		return &kwhmutating.MutatorResult{}, nil
+	}
+
+	if pod.Spec.SchedulerName != "sds-lvm" {
```

Comment:
Please change 'sds-lvm' to a constant.

Reply:
I think we need to mutate pods that either don't have a schedulerName or have the default one specified.